Pod/Classes/PINRemoteImageManager.m

```diff
@@ -1070,12 +1070,13 @@ - (void)didReceiveData:(NSData *)data forTask:(NSURLSessionDataTask *)dataTask
         __weak typeof(self) weakSelf = self;
         [_concurrentOperationQueue pin_addOperationWithQueuePriority:PINRemoteImageManagerPriorityLow block:^{
             typeof(self) strongSelf = weakSelf;
-            PINImage *progressImage = [progressiveImage currentImageBlurred:shouldBlur maxProgressiveRenderSize:maxProgressiveRenderSize];
+            CGFloat imageProgress = 1.0;
```

Comment:
Yeah I just meant that the task for this image could have a greater downloadProgress by now (just don't want it to be confused with progressDownload). If it makes sense to you I'll change it accordingly though